lib/hooks/use-user-info.ts

```diff
@@ -0,0 +1,18 @@
+import { useEffect, useMemo, useState } from 'react';
+import { Member, useEcomAPI } from '../ecom';
+
+export function useUserInfo() {
+    const api = useEcomAPI();
+
+    const [user, setUser] = useState<Member>();
+
+    const isUserLoggedIn = useMemo(() => api.isUserLoggedIn(), [api]);
+
+    useEffect(() => {
+        if (api.isUserLoggedIn()) {
+            api.getUser().then((u) => setUser(u));
+        }
+    }, [api]);
+
+    return { user, isUserLoggedIn };
```

Comment:
we can just return user info.. no info = not logged in

Reply:
not exaclty. we can immediatelly know if user is logged in, but get user information requires a call to API.

Reply:
oh.. I see. I thought `isUserLoggedIn` is an API call as well, but it's not

let's rename to just `isLoggedIn` (in returned object).. it's clear that it's user